src/panels/JobMonitor/ExecutionUnit.js

```diff
@@ -0,0 +1,49 @@
+import React                from 'react';
+import CollapsibleWidget    from 'paraviewweb/src/React/Widgets/CollapsibleWidget';
+import { formatTime }       from '../../utils/Format';
+import style                from 'HPCCloudStyle/JobMonitor.mcss';
+
+export default React.createClass({
+  displayName: 'ExecutionUnit',
+
+  propTypes: {
+    unit: React.PropTypes.object,
+  },
+
+
+  render() {
+    if (this.props.unit.log === undefined || this.props.unit.log.length === 0) {
+      return (
+        <section key={this.props.unit._id} className={ style.listItem }>
+          <strong className={ style.itemContent }>{this.props.unit.name.split('.').pop()}</strong>
+          <div className={ style.itemContent }>{this.props.unit.status}</div>
+        </section>
+      );
+    }
+
+    return (
+        <section key={this.props.unit._id} className={ style.logListItem }>
```

Comment:
`keys={}` should be where the element repeats [here](https://github.com/Kitware/HPCCloud/pull/257/commits/42765b06048d08149985c7e754d33ec407252338#diff-c489ca458627ef15cb8c294d34d6cd1aR62)